library/src/main/java/com/novoda/downloadmanager/DownloadBatchFactory.java

```diff
@@ -16,22 +17,21 @@ static DownloadBatch newInstance(Batch batch,
                                      DownloadsFilePersistence downloadsFilePersistence,
                                      CallbackThrottle callbackThrottle) {
         DownloadBatchTitle downloadBatchTitle = DownloadBatchTitleCreator.createFrom(batch);
-        List<String> fileUrls = batch.getFileUrls();
+        Map<DownloadFileId, String> fileUrls = batch.getFileUrls();
         List<DownloadFile> downloadFiles = new ArrayList<>(fileUrls.size());
         DownloadBatchId downloadBatchId = batch.getDownloadBatchId();
```

Comment:
this is duplicated

Reply:
Done in https://github.com/novoda/download-manager/pull/292/commits/060157c6268b94eca060f508a31509099b148d7a